### PR TITLE
move linebreaks method into resource.rb

### PR DESCRIPTION
**Pull request by regineheberlein**

### CODE REVIEW

File: spec/reports/aspace2alma/get_MARCxml_spec.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 require_relative '../../../reports/aspace2alma/get_MARCxml'
 require 'spec_helper.rb'
+require 'nokogiri'
```
Comment (maxkadel) [inline spec/reports/aspace2alma/get_MARCxml_spec.rb:5]: Do we still need to add nokogiri here?

Comment (regineheberlein) [reply spec/reports/aspace2alma/get_MARCxml_spec.rb:5]: Yes, line 70!

Comment (regineheberlein) [reply spec/reports/aspace2alma/get_MARCxml_spec.rb:5]: no, we don't!

**Review by maxkadel — APPROVED** [review]
Thanks @regineheberlein !

**Review by maxkadel — CHANGES_REQUESTED** [review]
Wait! I think we might still need nokogiri in get_MARCxml.rb, I think we just don't need it in the spec?

**Review by maxkadel — APPROVED** [review]
Thanks!

**Pull request merged**